datalad_container/extractors/metalad_container.py

```diff
@@ -0,0 +1,86 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
```

Comment:
IIUC it wasn't necessary (it worked) because it was being treated as an [Implicit Namespace](https://peps.python.org/pep-0420/)

But since that wasn't deliberate, I've added an __init__.py